pkg/operator/options/suite_test.go

```diff
@@ -271,12 +148,7 @@ var _ = Describe("Options", func() {
 				"--cluster-endpoint", "https://karpenter-000000000000.hcp.westus2.staging.azmk8s.io",
 				"--kubelet-bootstrap-token", "flag-bootstrap-token",
 			)
-			//Expect(err).To(HaveOccurred()) // TODO: Add back when karpenter-global-settings (and merge logic) are completely removed
-
-			// TODO: Remove below when karpenter-global-settings (and merge logic) are completely removed
-			Expect(err).ToNot(HaveOccurred())
-			ctx = settings.ToContext(ctx, &settings.Settings{})
-			Expect(func() { opts.MergeSettings(ctx) }).To(Panic())
+			Expect(err).To(HaveOccurred())
 		})
```

Comment:
The following two (that I can't comment on) would be:
```
			Expect(err).To(MatchError(ContainSubstring("not a valid clusterEndpoint URL")))
			Expect(err).To(MatchError(ContainSubstring("vm-memory-overhead-percent cannot be negative")))
```

Reply:
Update both, as well as the others you commented on.